### PR TITLE
Adding is_label option in Field to remove unknown label

### PR DESCRIPTION
**Pull request by GregorySenay**
Hi all,

I add in the Field and Vocal class, the option "is_label", to be able to remove the default <unk> token when you are creating the LABEL dictionary.

This avoid to have an additional <unk> label in the class list.

### CODE REVIEW

File: torchtext/vocab.py [hunk 1]
```diff
@@ -40,21 +40,38 @@ def __init__(self, counter, max_size=None, min_freq=1, specials=['<pad>'],
...
+            self.stoi.update({tok: i for i, tok in
                           enumerate(['<unk>'] + specials)})
-        self.itos = ['<unk>'] + specials
+            self.itos = ['<unk>'] + specials
 
-        counter.subtract({tok: counter[tok] for tok in ['<unk>'] + specials})
+            counter.subtract({tok: counter[tok] for tok in ['<unk>'] + specials})
+        
+        else:
+            counter.update(specials)
```
Comment (nelson-liu) [inline torchtext/vocab.py]: as it currently stands, this would add the `<pad>` token to your class list as well.

**Comment by nelson-liu** [comment]
I like the idea behind this, it'd be nice to get a class list out of a Field. Not sure if this is the best way to do it, though --- will think a bit more about it.

File: torchtext/vocab.py (hunk 1 above)
Comment (GregorySenay) [reply torchtext/vocab.py]: Okay in that case, with the is_label option I can also remove the "specials"

**Comment by jekbradbury** [comment]
Added a similar feature as a part of #107; let me know if that doesn't work for you.

**Comment by GregorySenay** [comment]
Great! it works like this for me.
In case for others, you have to call the LABEL like this:
LABEL = data.Field(sequential=False, **unk_token=None**)

**Pull request closed without merging**